docs/_pages/releases.md

```diff
@@ -14,6 +14,8 @@ sidebar:
 
 ### Fixes
 
+* Resolve an issue preventing HaveAccessModifier from correctly recognizing internal interfaces and enums - [#1741](https://github.com/fluentassertions/fluentassertions/issues/1741)
```

Comment:
❌ The release note should point to the PR
🔧 The common convention is to use back-ticks around code identifiers such as `HaveAccessModifier`